protos/sigstore_rekor.proto

```diff
@@ -128,3 +128,12 @@ message TransparencyLogEntry {
         // payload from other sources to verify the signature.
         bytes canonicalized_body = 7;
 }
+
+// The RekorBundle is the signed material used to produce the Signed Entry
+// Timestamp signature. See notes on the InclusionPromise above.
+message RekorBundle {
```

Comment:
Have we confirmed that when marshalled as json as canonicalized, this works as expected? Just want to make sure there's no issues with field naming.

Reply:
I can run some tests soon. No need to wait on this PR for the new release.